docs/release-notes.rst

```diff
@@ -10,6 +10,78 @@
+   --log --level <log_level> --color <on/off>`` to ``det master config set --log.level=<log_level>
+   --log.color=<on/off>``.
+
+**New Features**
+
+-  Experiments: Add an experiment continue feature via a CLI command ``det e continue
+   <experiment-id>``. This allows users to resume or recover training for an experiment whether it
+   previously succeeded or failed. This is limited to single-searcher experiments and using it may
+   prevent the user from replicating the continued experiment's results.
+
+-  Experiments: Add a ``log_policies`` configuration option to define actions when a trial's log
+   matches specified patterns.
+
+   -  The ``exclude_node`` action prevents a failed trial's restart attempts (due to its
+      max_restarts policy) from being scheduled on nodes with matched error logs. This is useful for
+      bypassing nodes with hardware issues like uncorrectable GPU ECC errors.
+
+   -  The ``cancel_retries`` action prevents a trial from restarting if a trial reports a log that
+      matches the pattern, even if it has remaining max_restarts. This avoids using resources for
+      retrying a trial that encounters certain failures that won't be fixed by retrying the trial,
+      such as CUDA memory issues. For details, visit :ref:`experiment-config-reference` and
+      :ref:`master-config-reference`.
+
+   This option is also configurable at the cluster or resource pool level via task container
+   defaults.
+
+-  Experiments: Add a new experiment config option ``log_policies`` to allow configuring policies to
+   take after a regex is matched. There are two action types a trial can be configured to take
+
+   -  ``exclude_node``: If a trial fails and restarts, the trial will not schedule on a node that
+      reported a log that matched the regex provided. This can be used to allow trials to avoid
+      being scheduled on nodes with certain hardware issues like uncorrectable gpu ECC errors.
+
+   -  ``cancel_retries``: If a trial reports a log that matches this pattern, the trial will not be
+      restarted. This is useful for certain errors that are not transient such as too large of a
+      model that causes a CUDA out of memory error.
+
+   This can also be configured on a cluster or per resource pool option through task container
+   defaults. Please see :ref:`experiment-config-reference` and :ref:`master-config-reference` for
+   more information.
+
+-  CLI: Add a new CLI command ``det e delete-tb-files [Experiment ID]`` to delete local TensorBoard
+   files associated to a given experiment.
+
+**Improvements**
+
+-  Update default environment images to Python 3.9 from Python 3.8.
+
+**Bug Fixes**
+
+-  Kubernetes: Support enabling and disabling agents to prevent Determined from scheduling jobs on
+   specific nodes.
+
+   Upgrading from a version before this feature to a version after this feature only on Kubernetes
```

Comment:
```suggestion
   On Kubernetes, upgrading from a version before this feature to a version after this feature
```

Reply:
this was my bad, I accidentally picked up an old release note

since it was an auto save emacs file that I have git ignored

Reply:
`backup-directory-alist` (https://www.gnu.org/software/emacs/manual/html_node/emacs/Backup.html) :wink: